src/tyr/route_serializer_osrm.cc

```diff
@@ -251,17 +251,17 @@ void route_summary(json::MapPtr& route,
                    const google::protobuf::RepeatedPtrField<valhalla::DirectionsLeg>& legs,
                    bool imperial) {
   // Compute total distance and duration
-  float duration = 0.0f;
-  float distance = 0.0f;
+  double duration = 0.0f;
+  double distance = 0.0f;
   for (const auto& leg : legs) {
     distance += leg.summary().length();
     duration += leg.summary().time();
   }
 
   // Convert distance to meters. Output distance and duration.
-  distance *= imperial ? 1609.34f : 1000.0f;
-  route->emplace("distance", json::fp_t{distance, 1});
-  route->emplace("duration", json::fp_t{duration, 1});
+  distance *= imperial ? 1609.344 : 1000.0;
+  route->emplace("distance", json::fp_t{distance, 3});
+  route->emplace("duration", json::fp_t{duration, 3});
```

Comment:
are milliseconds too fine a measure?

Reply:
:+1: milliseconds makes sense to me to minimize accumulated error